Cargo.toml

```
@@ -6,7 +6,7 @@ repository = "https://github.com/tari-project/randomx-rs"
 homepage = "https://tari.com"
 readme = "README.md"
 license = "BSD-3-Clause"
-version = "1.1.10"
+version = "1.1.12"
```

You could do this if you wanted to track the RandomX version 

```suggestion
version = "1.1.12+1.1.10"
```

---

Lol, it was naive to think I could keep the versions the same